Pilot1/P1B2/pytorch_utils.py

```diff
@@ -0,0 +1,140 @@
+from __future__ import absolute_import
+
+import torch
+import torch.nn
+import torch.nn.init
+import torch.optim
+import torch.nn.functional as F
+
+from default_utils import set_seed as set_seed_defaultUtils
+
+def set_parallelism_threads(): # for compatibility
+    pass
+
+def set_seed(seed):
```

Comment:
I guess similar functions are already defined in `common/pytorch_utils.py` please use the common file. If there is anything we need to add, let me know

Reply:
The pytorch versions of build_optimizer etc may not exist yet, this is good to have. I can look at uplifting these to a common/pytorch_utils.py file.

Reply:
I was wrong, these seem to be copies of existing common/pytorch_utils.py? Are there additions made?

Reply:
I have mirrored the changes from my local pytorch_utils.py within P1B2 to the common/pytorch_utils.py. And deleted the local pytorch_utils.py within P1B2.